README.md

```diff
@@ -209,19 +129,14 @@ If you like this project give me ⭐️
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
 I would like to thank Microverse
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 [MIT](https://mit-license.org/)
```

Comment:
- [ ] (optional) Instead of putting a link to the website for the MIT license template, It would be better if you created your own file and then added the template edited with your credentials for q more professional readme file.